apps/api/src/view-models/edgeConnectionViewModels/note.view-model.ts

```diff
@@ -27,8 +30,10 @@ export class NoteViewModel extends BaseViewModel implements INoteViewModel {
     })
     readonly connectedResources: EdgeConnectionMember[] = [];
 
-    constructor({ id, note, members, connectionType }: EdgeConnection) {
-        super({ id });
+    constructor(edgeConnection: EdgeConnection) {
+        super(edgeConnection);
+
+        const { note, members, connectionType } = edgeConnection;
```

Comment:
Should the `note` property be `MultilingualText`?

Reply:
That could be a good idea